metro.config.js

```diff
@@ -1,6 +1,9 @@
 const { getDefaultConfig } = require('expo/metro-config');
 const { withNativeWind } = require('nativewind/metro');
 
+const defaultConfig = getDefaultConfig(__dirname);
+
+module.exports = withNativeWind(defaultConfig, { input: './global.css' });
 const config = getDefaultConfig(__dirname);
 
 module.exports = withNativeWind(config, { input: './global.css' });
```

Comment:
Remove this, its doing the same thing as the one above

Reply:
**Done**!